### PR TITLE
IOS 7154 global disabled state definition

### PR DESCRIPTION
**Pull request by idenjoe**
🎟️ Jira ticket

[IOS-7154](https://jira.tid.es/browse/IOS-7154) 

🥅 What's the goal?

- We made a global definition for disabled states in Mística: https://www.figma.com/file/dvMqeg6s8tDh9Hyxu7IYDl/%F0%9F%94%B8-Subatomic-Components?node-id=102%3A1071
- Small and link buttons paddings
- Corner Radius for several components
- Feedback and empty state description
- Empty State card icon size

🚧 How do we do it?

- Removed all disabled tags
- Modified Button and InputField to set alpha to 50% when the component is disabled.
- Modified horizontal padding for small and link buttons to 12
- Changed cornerRadius from 4 to 8 in the following components:
      - Cards
      - Callouts
      - Popovers
      - Boxed lists
      - Empty State Card
      - Forms
- Feedback subtitle changed from textPreset4 light to textPreset3 regular.
- Changed Empty State card icon size to 48.

🧪 How can I verify this?

### CODE REVIEW

**Comment by yceballost** [comment]
can you export a new Mística Catalog update? 🥺

**Review by yceballost — APPROVED** [review]
fantastic!

**Review by jmbrocal — APPROVED** [review]

**Review by jmbrocal — APPROVED** [review]

File: Mistica/Source/Components/InputField/InputField.swift [hunk 1]
```diff
@@ -96,7 +99,7 @@ public class InputField: UIView {
         let view = UIView()
         view.layer.borderWidth = 1
         view.layer.borderColor = UIColor.border.cgColor
-        view.layer.cornerRadius = 4
+        view.layer.cornerRadius = 8
```
Comment (DavidMarinCalleja) [inline Mistica/Source/Components/InputField/InputField.swift:102]: we have a utils method maybe we could use that method and update the value, WDYT?
https://github.com/Telefonica/mistica-ios/blob/1b0dfd14b9100ed30d05c62e7a0c3f3677a15e67/Mistica/Source/Utils/Extensions/UIView%2BUtils.swift#L240

Comment (jmpg93) [reply Mistica/Source/Components/InputField/InputField.swift:102]: Sure, I have to open another PR. I will add the change there. Thx!

**Review by DavidMarinCalleja — APPROVED** [review]

**Pull request merged**